requirements.txt

```diff
@@ -0,0 +1,7 @@
+PyGithub==1.56
```

Comment:
Over time, especially as we set up e2e tests, we are likely to have different requirements files for different purposes. This one should probably move to hack as something like `hack/requirements.txt` so we can differentiate it from `validate-microshift/requirements.txt` in the future.

Reply:
I think that `script/auto-rebase/rebase.py` and (if we decide so) `e2e/tests.py` should have separate `requirements.txt` in their own dirs. Does it make sense from how python project operate?

Reply:
I am unsure what is the recommended way, however we need all the dependencies installed before running pylint. We may have to loop over all the `requirements.txt` files if we choose to have multiple.